docs/readme-nuttx.md

```diff
@@ -3,8 +3,8 @@
 ## Description
 
 The NuttX port of MCUboot secure boot library expects that the platform provides a Flash storage with the following partitions:
-- `CONFIG_MCUBOOT_PRIMARY_SLOT_PATH`: MTD partition for the application firmware image PRIMARY slot;
-- `CONFIG_MCUBOOT_SECONDARY_SLOT_PATH`: MTD partition for the application firmware image SECONDARY slot;
+- `CONFIG_MCUBOOT_SLOT_PRIMARY_PATH`: MTD partition for the application firmware image PRIMARY slot;
+- `CONFIG_MCUBOOT_SLOT_SECONDARY_PATH`: MTD partition for the application firmware image SECONDARY slot;
```

Comment:
this would be a breaking change so next release would need major version increment @d3zd3z